### ISSUE TITLE
Cannot make it update apache logs

### ISSUE THREAD

**Issue by rvencu** [axkibe/lsyncd#513]
I added inotifyMode = "Modify" in settings but for some reason I still cannot make lsyncd 2.2.2 to update apache log files at destination.

When I restart the service after conf file editing, there is a first transfer, then transfer stops.

I have a second sync that detects closing (new?) files from another folders and that one still works after changing the inotifyMode.

**Comment by axkibe** [comment]
Are the log files possibly on a network drive?

Otherwise please provide a log with -log all (or at least -log Inotify)

**Comment by rvencu** [comment]
Just a simple single disk VM. I am having a hard time looking for settings to expand the log as suggested

**Comment by rvencu** [comment]
Since the apache log is changing constantly would not be better to launch rsync from a cron job at every 10 minutes instead of relying in inotify? Still not found how to enable log all in lsyncd.conf

**Comment by axkibe** [comment]
You can't do it in config file, sorry this has been an oversight which I'll fix in future.

You put ``--log all``` as command line parameter.

If cron job every x mintues works for you, go for it. I really disliked the idea to have something constantly running which in my use cases mostly would have done nothing.

**Comment by rvencu** [comment]
Yes. At the target there is a web analytics app that likes to display "Real Time" stats so maybe I need to run the cron even at minute! This will also minimize data loss when the log is rotated once per day.
